pepper/libpepper.py

```diff
@@ -307,7 +307,7 @@ def low(self, lowstate, path='/'):
         '''
         return self.req(path, lowstate)
 
-    def local(self, tgt, fun, arg=None, kwarg=None, expr_form='glob',
+    def local(self, tgt, fun, arg=None, kwarg=None, tgt_type='glob',
```

Comment:
Instead of doing this, can you still accept expr_form and just assign it to tgt_type.

I don't want to break peoples scripts that use expr_form.